.jenkins/infrastructure/build_docker_images.Jenkinsfile

```diff
@@ -1,70 +1,36 @@
-                puboeDeploy = oe.dockerImage("oeciteam/oetools-deploy:${DOCKER_TAG}", ".jenkins/infrastructure/dockerfiles/linux/Dockerfile.deploy", buildArgs)
-            }
-            stage("Push to OE Docker Registry") {
-                docker.withRegistry(OETOOLS_REPO, OETOOLS_REPO_CREDENTIAL_ID) {
-                    oe.exec_with_retry { oefull1804.push() }
-                    oe.exec_with_retry { oefull2004.push() }
-                    oe.exec_with_retry { oeminimal1804.push() }
-                    oe.exec_with_retry { oeDeploy.push() }
-                    if(TAG_LATEST == "true") {
-                        oe.exec_with_retry { oefull1804.push('latest') }
-                        oe.exec_with_retry { oefull2004.push('latest') }
-                        oe.exec_with_retry { oeminimal1804.push('latest') }
-                        oe.exec_with_retry { oeDeploy.push('latest') }
-                    }
-                }
-            }
-            stage("Push to OE Docker Hub Registry") {
-                docker.withRegistry('', OETOOLS_DOCKERHUB_REPO_CREDENTIAL_ID) {
-                    if(TAG_LATEST == "true") {
-                        oe.exec_with_retry { puboefull1804.push() }
-                        oe.exec_with_retry { puboefull2004.push() }
-                        oe.exec_with_retry { puboeminimal1804.push() }
-                        oe.exec_with_retry { puboeDeploy.push() }
-                        oe.exec_with_retry { puboefull1804.push('latest') }
-                        oe.exec_with_retry { puboefull2004.push('latest') }
-                        oe.exec_with_retry { puboeminimal1804.push('latest') }
-                        oe.exec_with_retry { puboeDeploy.push('latest') }
-                    }
-                }
-            }
+parallel "Windows Stage": {
+        stage("Build Windows Docker Containers") {
+          build job: '/CI-CD_Infrastructure/Windows Docker Container Build',
+          parameters: [string(name: 'REPOSITORY_NAME', value: env.REPOSITORY_NAME),
+                       string(name: 'BRANCH_NAME', value: env.BRANCH_NAME),
+                       string(name: 'INTERNAL_REPO', value: OETOOLS_REPO),
+                       string(name: 'INTERNAL_REPO_CREDS', value: OETOOLS_REPO_CREDENTIAL_ID),
+                       string(name: 'DOCKERHUB_REPO_CREDS', value: OETOOLS_DOCKERHUB_REPO_CREDENTIAL_ID),
+                       string(name: 'DOCKER_TAG', value: env.DOCKER_TAG),
+                       string(name: 'AGENTS_LABEL', value: env.WINDOWS_AGENTS_LABEL),
+                       string(name: 'OECI_LIB_VERSION', value: OECI_LIB_VERSION),
+                       booleanParam(name: 'TAG_LATEST', value: env.TAG_LATEST)]
+        }
+    }, "Linux Stage": {
+        stage("Build Linux Docker Containers") {
+          build job: '/CI-CD_Infrastructure/Linux Docker Container Build',
```

Comment:
Would you mind if we did

"Linux Docker Container Build" -> "Linux-Docker-Container-Build" and for windows as well?

Reply:
Sure thing; updating the Jenkins job names now